osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_body.txt

```diff
@@ -1,10 +1,14 @@
-{% load navigation %}
-The extraction order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}" has been processed.
-{% if successful_exports.count > 0 %}Results available for download:
-{% for export in successful_exports %}- {{ export.get_file_format_display }}{% if export.output_file.file %} ({{ export.output_file.file.url | siteabsoluteurl:request }}){% endif %}
+{% load navigation %}This is an automated email from the OSMaxx service.
```

Comment:
Yes it has to do with testing: it breaks the tests, because we test this line as well.

The test states, that the first line needs to be `This is an automated email from OSMaxx.`. You changed that line to `This is an automated email from the OSMaxx service.` in the template, now the test for this template does fail -> you need to adjust the test as well.